blocks/table/table.css

```diff
@@ -0,0 +1,134 @@
+/* stylelint-disable no-descending-specificity */
+.table-container {
+    margin-bottom: 32px;
+}
+
+.table-container > div.table-wrapper {
+    margin-bottom: 0;
+}
+
+.table-container > .table-wrapper + .table-wrapper {
+    margin-top: 0;
+}
+
+.table-form-submit-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.table-form-submit-wrapper button {
+    height: 50px;
+    width: 300px;
+    background-color: var(--blue);
+    color: var(--white);
+    border-radius: 6px;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-s);
+    margin: 20px 0;
+}
+
+.table-form-submit-wrapper button:disabled {
+    background-color: var(--blue-light);
+}
```

Comment:
not in example page, not scoped.

Reply:
I removed this section of code but I don't know what the right answer is here and could use your advice. The button is a part of a form that wraps the table for wholesale. This table should not have anything other than table code in it but where do I style things if they don't have a .css file? Just inline?